test/Dataflow/demanded-bits5-invalid.opt

```diff
@@ -1,6 +1,6 @@
 
 
-; RUN: %souper-check -infer-rhs -souper-only-infer-iN %s > %t 2>&1
+; RUN: %souper-check -souper-double-check -infer-rhs -souper-only-infer-iN %s > %t 2>&1
```

Comment:
please don't delete the existing RUN: lines unless there's a good reason to do so

Reply:
double check just adds an extra check, not replacing anything existing.
If this test succeeds, the previous one would have succeeded.